include/oneapi/dpl/pstl/utils.h

```diff
@@ -302,8 +302,8 @@ class __transform_if_unary_functor
     mutable _UnaryPred _M_pred;
 
   public:
-    explicit __transform_if_unary_functor(_UnaryOper&& __op, _UnaryPred&& __pred)
-        : _M_oper(::std::forward<_UnaryOper>(__op)), _M_pred(::std::forward<_UnaryPred>(__pred))
+    explicit __transform_if_unary_functor(_UnaryOper __op, _UnaryPred __pred)
```

Comment:
I agree this is incorrect as it was written before this PR.
Would a better fix be to make this constructor templated with its own local universal references, and keep the forward?

We already explicitly use explicit template parameters whenever we instantiate these types.

Reply:
> I agree this is incorrect as it was written before this PR. Would a better fix be to make this constructor templated with its own local universal references, and keep the forward?
> 
> We already explicitly use explicit template parameters whenever we instantiate these types.

We discussed with @rarutyun about this implementations and he proposed do not use `&&` in constructor params of these functors.